.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+
+updates:
+  - package-ecosystem: 'cargo'
+    directory: '/local-registry'
+    schedule:
+      interval: 'weekly'
+    groups:
+      - all:
+        patterns:
+         - "*"
```

Comment:
This should reduce the noise, maximum 1 PR per week. I see no reason to update separate crate versions in individual PRs.